csharp/src/Google.Protobuf.Test.TestProtos/Google.Protobuf.Test.TestProtos.csproj

```diff
@@ -14,6 +14,10 @@
     <IsPackable>False</IsPackable>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.0' ">

```

Comment:
I'm actually somewhat surprised that all the generated code still builds despite the `<LangVersion>3.0</LangVersion>` being set for all the target frameworks (see above).

I thought it wouldn't be possible to reference the `ref structs` from a C# 3.0 code. It that works, that's a good kind of surprise (we want the generated code to be compilable with old compiler), but I didn't expect that.

Reply:
I'm surprised too. Perhaps it is just defining new `ref structs` that is limited to 7.2+.

Reply:
Structs and passing structs by ref has been supported forever so 🤷‍♂